### PR TITLE
fixes #11122 - initialize ENCRYPTION_PREFIX only if it is not already initialized

### PR DESCRIPTION
**Pull request by shlomizadok**

### CODE REVIEW

**Comment by lzap** [comment]
_Thank you_ :-)

**Comment by ohadlevy** [comment]
any idea why this file was loaded multiple times?

**Comment by domcleal** [comment]
It's not loaded multiple times, the concern is included multiple times.

**Comment by domcleal** [comment]
I suppose the constant is actually defined in Encryptable, so perhaps could be hoisted out of the included block.

**Comment by shlomizadok** [comment]
Moved outside the block. Seems more reasonable.

File: app/models/concerns/encryptable.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 module Encryptable
   extend ActiveSupport::Concern
 
+  ENCRYPTION_PREFIX = "encrypted-"
+
```
Comment (ohadlevy) [inline app/models/concerns/encryptable.rb]: Trailing whitespace

**Comment by shlomizadok** [comment]
Rubocop fixed

**Comment by domcleal** [comment]
Merged as b131eda, thanks @shlomizadok.

**Pull request closed without merging**